### DISCUSSION TITLE
Extend track editor : Add a button to go to the last/first track piece.

### DISCUSSION THREAD

**Discussion by mathfournier99** [OpenRCT2/OpenRCT2#20432] [Feature requests] [answered]
### Problem : 
Sometimes I want to edit the back of my coaster tracks and going from the first piece to the last one requires many clicks on the "next track piece button". 

### Proposed solution : 
Add a button to instantly go to the first or last track piece. (See image)
![image](https://github.com/OpenRCT2/OpenRCT2/assets/65820162/81b52df8-4fa5-4708-8e28-840631154850)

Note : 
When the track is a single piece or a closed circuit, there is no more a first or last track piece. 
In those cases we disable the buttons.

Thanks (:

**Comment by Broxzier** [comment, marked as answer]
I assume you already know this, but you can just right-click a track piece to instantly jump to the section you want to edit.

As for these new buttons, I'm not against them, but I do wonder if it's worth it. I can already see this causing issues when multiple people are working on the same ride, or a track piece is removed, and cause infinite loops on merged tracks. It won't be trivial to make this work in all cases.

**Reply by mathfournier99** [reply]
Wrong assumption :P.
I didn't knew about right clicking. I started playing this week so I still have so much to learn (: 
Right clicking fixes my issue and is even more convenient than what I proposed. 
We can close this feature request. 
Thanks a lot
